### PR TITLE
Rename metadata dictionary

### PR DESCRIPTION
**Pull request by tguilbert-google**
Rename VideoFrameMetadata to VideoFrameCallbackMetadata to avoid a naming collision with WebCodec's VideoFrameMetadata. The dictionary names are not exposed to JS, so the verbosity is not important

### CODE REVIEW

**Comment by tguilbert-google** [comment]
FYI, @youennf

File: index.bs [hunk 1]
```diff
@@ -59,13 +59,13 @@ discontinuity in the {{presentedFrames}}).
 Note: A web author could know if a callback is late by checking whether {{expectedDisplayTime}} is equal
 to *now*, as opposed to roughly one v-sync in the future.
 
-The {{VideoFrameRequestCallback}} also provides useful {{VideoFrameMetadata|metadata}} about the video
+The {{VideoFrameRequestCallback}} also provides useful {{VideoFrameCallbackMetadata|metadata}} about the video
 frame that was most recently presented for composition, which can be used for automated metrics analysis.
 
-# VideoFrameMetadata #    {#video-frame-metadata}
+# VideoFrameCallbackMetadata #    {#video-frame-metadata-callback}
```
Comment (youennf) [inline index.bs]: Should it be video-frame-callback-metadata?
Ditto for other links below

**Review by youennf — APPROVED** [review]

File: index.bs (hunk 1 above)
Comment (tguilbert-google) [reply index.bs]: Yes, thanks for catching it!

**Comment by tguilbert-google** [comment]
@tidoust or @cwilso, I'm having trouble importing WICG/video-rvfc in the W3 repo tool.

This repo was renamed from WICG/video-raf (which is still listed as a valid repo). Is this a permission issue, or is the rename causing issues? Thanks!

**Comment by tidoust** [comment]
> @tidoust or @cwilso, I'm having trouble importing WICG/video-rvfc in the W3 repo tool.
> 
> This repo was renamed from WICG/video-raf (which is still listed as a valid repo). Is this a permission issue, or is the rename causing issues? Thanks!

@deniak @dontcallmedom, when I try to [import](https://labs.w3.org/repo-manager/repo/import) the `WICG/video-rvfc` repository into the repository manager, I get an error rendered as "e.message is undefined" (which isn't super useful ;)). Could you look into it?

**Comment by deniak** [comment]
@tidoust I manually renamed the repo in the repository manager although renaming should be handled automatically.
I'll investigate but in the meantime, PRs from WICG/video-rvfc should now validate.

**Comment by deniak** [comment]
Do you know when the repository was renamed? The webhook history page only shows recent payloads and the organization audit log page seems broken at the moment.

**Comment by tguilbert-google** [comment]
> Do you know when the repository was renamed? The webhook history page only shows recent payloads and the organization audit log page seems broken at the moment.

I think it was renamed somewhere during April 2020, based off of [this comment](https://github.com/WICG/video-rvfc/issues/44#issuecomment-612148534).

Thanks for manually updating the tool!

**Comment by deniak** [comment]
> > Do you know when the repository was renamed? The webhook history page only shows recent payloads and the organization audit log page seems broken at the moment.
> 
> I think it was renamed somewhere during April 2020, based off of [this comment](https://github.com/WICG/video-rvfc/issues/44#issuecomment-612148534).
> 

The repository manager started to handle repository renaming in [September 2021](https://github.com/w3c/ash-nazg/pull/223) which explains why it still had the old name.
Anyhow, the repository manager has the right name for that repo now.

**Comment by tguilbert-google** [comment]
Thanks for the info!

**Pull request merged**